packages/react-core/src/components/CodeBlock/__tests__/CodeBlock.test.tsx

```diff
@@ -1,30 +1,32 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import { CodeBlock } from '../CodeBlock';
 import { CodeBlockAction } from '../CodeBlockAction';
 import { CodeBlockCode } from '../CodeBlockCode';
 
-test('CodeBlock renders successfully', () => {
-  const { asFragment } = render(<CodeBlock>test text</CodeBlock>);
-  expect(asFragment()).toMatchSnapshot();
-});
+test('CodeBlock renders', () => {
+    render(<CodeBlock>test text</CodeBlock>);
+    expect(screen.getByText('test text')).toBeVisible();
+  });
 
-test('CodeBlockAction renders successfully', () => {
-  const { asFragment } = render(<CodeBlockAction>action</CodeBlockAction>);
-  expect(asFragment()).toMatchSnapshot();
-});
+test('CodeBlock content renders with class pf-v5-c-code-block__content by default', () => {
```

Comment:
Can you follow what Adam is doing in his [PR](https://github.com/patternfly/patternfly-react/pull/9266/files) for updating our test to not have hard coded styles.  This will make it easier when updating versions of PF styles in the future.